### PR TITLE
Fix documentation example

### PR DESCRIPTION
**Pull request by RazrFalcon**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -33,7 +33,8 @@
 //! let ref mut w = BufWriter::new(file);
 //!
 //! let mut encoder = png::Encoder::new(w, 2, 1); // Width is 2 pixels and height is 1.
-//! encoder.set(png::ColorType::RGBA).set(png::BitDepth::Eight);
+//! encoder.set_color(png::ColorType::RGBA);
+//! encoder.set_depth(png::BitDepth::Eight);
```
Comment (HeroicKatora) [inline src/lib.rs]: This example is marked `ignore`, that's why CI hasn't caught it before. Replacing this with `no_run` might be better.

**Comment by RazrFalcon** [comment]
@HeroicKatora Looks like it was disabled because `Encoder` is optional

**Comment by HeroicKatora** [comment]
Two possible solutions:

* Add a guard within the example, e.g. wrap it in a block marked`#[cfg(feature="png-encoding")]`.
* Make the annotation (`no_run` vs. `ignore`) depend on the feature configuration. This is cleaner, and I've seen the trick myself only recently. But you can use `cfg_attr` to insert parts of the documentation. So you start the doc-test in two different manners depending on configuration:
  ```
  #[cfg_attr(feature = "png-encoding", doc = "```no_run")]
  #[cfg_attr(not(feature = "png-encoding"), doc = "```ignore")]
  ```

**Comment by RazrFalcon** [comment]
Done, but looks like it was already fixed in the master.

**Comment by HeroicKatora** [comment]
Huh. Sorry then @RazrFalcon

**Pull request closed without merging**